andrew-oyanguren/typescript-portfolio
# PR-37 Image Carousel Auto-Cycle

### Summary

This work adds an initial auto cycle of the image carousel before showing controls. Also, various css updated on large devices throughout the app.

### Changes

- New effect to handle auto next().
- Various CSS improvements.

### Testing Guide

1. `npm run dev`
2. The home page should load; after 4 seconds should start auto-cycle through images; followed by showing controls.
